text/0000-rfc-process.md

```diff
@@ -0,0 +1,238 @@
+
+**Steward**: The steward of an RFC is responsible for shepherding an RFC through the process, including working with the author to ensure RFC completeness and quality, and building consensus among stakeholders.
+
+**Call for Votes**: When an RFC is deemed ready by a team lead, that team lead initiates the voting process with a call for votes. At this point the RFC is closed to modification.
+
+**End Date**: When a call for votes is initiated, an end date for voting is set. Any person wishing to vote on an RFC must do so by the end date.
+
+**Voting Window**: The time period between a call for votes and the voting end date is referred to as the voting window.
+
+**Lazy Consensus**: Voting on project level RFCs is done by lazy consensus. Any project member with a binding vote who has not voted by the end date is assumed to assent to the RFC.
+
+**Binding Vote**: Binding Votes have formal power within the RFC process. A single no vote by a project member with a binding vote is sufficient to reject an RFC.
+
+**Non-Binding Vote**: Anyone from the project or community may cast a non-binding vote on an RFC to express their position. While these votes do not have formal power they are taken seriously and may affect the votes of members with binding votes.
+
+
+# Motivation
+[motivation]: #motivation
+
+This RFC process is an evolution of our [previous process](0004-rfc-process.md).
+
+As the project grows our process needs to grow with it. Compared to the previous iteration, this new process should:
+1. **Re-balance responsibilities**. We wish to empower team leads to take ownership of the future of the project and transition the TOC into an oversight role.
+2. **Favor action**. By moving from a super-majority vote to a lazy-consensus we remove the ability to block change through inaction. It is now the responsibility of those with concerns to raise them in a timely fashion.
+3. **Manage our bandwidth**. Sometimes we need to say "not now" to a good idea to manage the number of large changes proceeding simultaneously. This new process should help the project focus on shipping high-quality enhancements by managing the number of in-flight RFCs, and ensuring each approved RFC has an implementer.
+4. **Improve Readability**. By allowing amendments to RFCs we make life easier for anyone wishing to understand a change by ensuring they can find the full plan in a single document.
+
+# What it is
+[what-it-is]: #what-it-is
+
+### What Types of Changes Require an RFC?
+
+Any "substantive" change to the project requires an RFC. substantive includes but is not limited to:
+* changes to the specification.
+* the adoption, creation, or deprecation of a component (e.g. a new platform implementation, a new shared library, a new system buildpack). 
+* new features (e.g. a new pack command, a new flag on an existing pack command, an addition to the buildpack API)
+* any major refactor that affects consumers of our libraries or materially impacts contribution.
```

Comment:
We typically haven't followed this. Is this something we really want/need?

Reply:
https://github.com/buildpacks/rfcs/blob/main/text/0082-pack-package-refactor.md is one example but I recall at the time we thought it might have been better as a sub-team RFC.

Reply:
I would have expected ["pack package refactor"](https://github.com/buildpacks/rfcs/blob/main/text/0082-pack-package-refactor.md) to be a sub-team RFC if we were practicing them at that time.

> * any major refactor that affects consumers of our libraries or materially impacts contribution.

"major" feels subjective to me. I would rather want to see maintainers and leads make a judgement call on what, if any, refactors should be RFCs.

Reply:
From 9/1 working group:

* "major" is intentionally subjective to allow maintainers and leads to make that judgment call.